src/rhsmlib/services/entitlement.py

```diff
@@ -375,3 +375,7 @@ def remove_entitlements_by_serials(self, serials):
         self.entcertlib.update()
 
         return removed_serials, unremoved_serials
+
+    def reload(self):
```

Comment:
If this method is only used in the EntitlementDbusObject.reload, why not include this logic there?

Reply:
I remember there being a reason, I believe it's related to there already being an EntitlementService object in objects/entitlement and singletons made by injection might make that weird, but I don't know if that hinders me moving that logic to objects/entitlement. I will ask @kahowell

Reply:
The logic could be moved to EntitlementDBusObject reload but I think I will leave the logic where it is since we may not always use a dbus and I do not think it will make the code much more readable if I put in EntitlementDBusObject as opposed to where it is now